source/v4/users.yaml

```diff
@@ -122,4 +122,29 @@
           $ref: '#/responses/BadRequest'
         '401':
           $ref: '#/responses/Unauthorized'
+    delete:
+      tags:
+        - users
+      summary: Deactivate a user account.
+      description: |
+        Deactivates the user by archiving its user object.
+        ##### Permissions
+        Must be logged in as the user being updated or have the `edit_other_users` permission.
+      parameters:
+        - name: user_id
+          in: path
+          description: User GUID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: User deactivation successful
+          schema:
+            $ref: '#/definitions/StatusOK'
+        '400':
+          $ref: '#/responses/BadRequest'
+        '401':
+          $ref: '#/responses/Unauthorized'
```

Comment:
Updated! I didn't notice that I had put a forbidden request on the unit test. Thanks for catching that!